nft/Cargo.toml

```
@@ -0,0 +1,17 @@
+[project]
+name = "NFT"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels = "0.14"
+fuels-abigen-macro = "0.14"
+tokio = { version = "1.12", features = ["rt", "macros"] }
+fuel-tx = "0.12.1"
```

Use major.minor instead of major.minor.patch for deps, so that you can automatically pick up patches.
```suggestion
fuel-tx = "0.12"
```